### PR TITLE
tiledb has been stable recently

### PR DESCRIPTION
**Pull request by CJ-Wright**
Closes #489 
Closes #495 
Closes #507

### CODE REVIEW

**Comment by isuruf** [comment]
Wouldn't you need a migration or a hotfix?

**Comment by CJ-Wright** [comment]
I don't know, maybe? I'll draw up the migration.

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -303,7 +303,7 @@ pin_run_as_build:
   tk:
     max_pin: x.x
   tiledb:
-    max_pin: x.x
+    max_pin: x.x.x
```
Comment (isuruf) [inline recipe/conda_build_config.yaml]: You can keep this `x.x`

File: recipe/migrations/tiledb_looser.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  build_number:
+    1
+
+# The name of the feedstock you wish to migrate with dashes replaced by
+# underscores
+pin_run_as_build:
+  max_pin:
+    tiledb: x.x
+tiledb:
+  - 1.70    # new version to build against
```
Comment (isuruf) [inline recipe/migrations/tiledb_looser.yaml:26]:
```suggestion
  - 1.7    # new version to build against
```

**Comment by CJ-Wright** [comment]
Would `Rebuild for looser_tiledb` be better?

**Comment by isuruf** [comment]
`tiledb_loosen_pin` ?

**Comment by CJ-Wright** [comment]
works for me

**Comment by isuruf** [comment]
We should fix the run_exports at https://github.com/conda-forge/tiledb-feedstock/blob/master/recipe/meta.yaml#L20

**Comment by CJ-Wright** [comment]
done in https://github.com/conda-forge/tiledb-feedstock/pull/60

**Comment by CJ-Wright** [comment]
@isuruf is this ok to go in?

**Comment by jakirkham** [comment]
cc @conda-forge/tiledb

**Pull request merged**